rcl/src/rcl/time.c

```diff
@@ -72,6 +72,21 @@ rcl_get_ros_time(void * data, rcl_time_point_value_t * current_time)
   return RCL_RET_OK;
 }
 
+bool
+rcl_time_point_value_valid(rcl_time_point_value_t time_point_value)
+{
+  return time_point_value > 0;
+}
+
+bool
+rcl_time_point_valid(rcl_time_point_t * time_point)
```

Comment:
even if `rcl_clock_type_t` is `RCL_CLOCK_UNINITIALIZED`, this will return true if `time_point->nanoseconds` is greater than zero.